longshot/application/main.tf

```diff
@@ -0,0 +1,218 @@
+variable "pipeline" {}
+variable "pipeline_stage" {}
+variable "name" {}
+variable "host" {}
+
+variable "web_size" {
+  default = "Standard-1X"
+}
+
+variable "web_scale" {
+  default = "1"
+}
+
+variable "queue_size" {
+  default = "Standard-1X"
+}
+
+variable "queue_scale" {
+  default = "1"
+}
+
+variable "redis_type" {
+  default = "hobby-dev"
+}
+
+variable "database_name" {}
+
+variable "database_type" {
+  default = "db.t2.medium"
+}
+
+variable "database_size" {
+  default     = 100
+  description = "The amount of storage to allocate to the database, in GB."
+}
+
+variable "email_name" {}
+variable "email_address" {}
+
+variable "papertrail_destination" {}
+
+data "aws_ssm_parameter" "database_username" {
+  name = "/${var.name}/rds/username"
+}
+
+data "aws_ssm_parameter" "database_password" {
+  name = "/${var.name}/rds/password"
+}
+
+data "aws_ssm_parameter" "mandrill_api_key" {
+  name = "/mandrill/api-key"
+}
+
+resource "heroku_app" "app" {
+  name   = "dosomething-${var.name}"
+  region = "us"
+
+  organization {
+    name = "dosomething"
+  }
+
+  config_vars {
+    # App settings:
+    APP_ENV   = "production"
```

Comment:
env should be a parameter

Reply:
We've tended to just set this to `production` across the board, but definitely could be useful to use actual environment names here. I've updated this in #55.